src/compat/libc/stdlib/mbtowc.c

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file
+ * @brief mbtowc function.
+ *
+ * @see stdlib.h
+ *
+ * @date 01.11.18
+ * @author Chubukov Filipp
+ */
+
+#include <stdlib.h>
+#include <string.h>
+#include <defines/size_t.h>
+#include <defines/wchar_t.h>
+
+int mbtowc(wchar_t *out, const char *in, size_t n) {
```

Comment:
It looks like this function does the same stuff as `mbstowcs()`, but that's incorrect, it should write at most 1 wchar to `out`

```
mbtowc   - convert a multibyte sequence to a wide character
mbstowcs - convert a multibyte string to a wide-character string 
```

Reply:
changed function